query-engine/driver-adapters/src/conversion/js_arg_type.rs

```diff
@@ -0,0 +1,33 @@
+#[derive(Debug, PartialEq)]
+pub enum JSArgType {
+    Int32,
+    Int64,
+    Float,
+    Double,
+    Boolean,
+}
```

Comment:
We can add more input `JSArgType` values in the future, as needed, for custom data manipulation on the adapters' side before feeding the query to the database.

Reply:
Can we expand the type to cover all `QuaintValue` already? It is really weird to get `null` as an arg type for string values

Reply:
Sure, done.